extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -175,12 +191,51 @@ export class AmpStoryReaction extends AMP.BaseElement {
   }
 
   /**
-   * Generates the template in rootEl_ and fills up with options.
-   * @param {!Element} unusedElement
+   * Reads the element attributes prefixed with option- and returns them as a list.
+   * eg: [
+   *      {optionIndex: 0, text: 'Koala'},
+   *      {optionIndex: 1, text: 'Developers', correct: ''}
+   *    ]
+   * @protected
+   * @return {?Array<!OptionConfigType>}
+   */
+  parseOptions_() {
+    const options = [];
+    toArray(this.element.attributes).forEach((attr) => {
+      // Match 'option-#-type' (eg: option-1-text, option-2-image, option-3-correct...)
+      if (attr.name.match(/^option-\d+-\w+$/)) {
+        const splitParts = attr.name.split('-');
+        const optionNumber = parseInt(splitParts[1], 10);
+        // Add all options in order on the array with correct index.
+        while (options.length < optionNumber) {
+          options.push({'optionIndex': options.length});
+        }
+        options[optionNumber - 1][splitParts[2]] = attr.value;
+      }
+    });
+    if (
```

Comment:
Nit: let's add a `devAssert(this.optionBounds_)` so anyone extending this abstract class knows about it